bazel/repositories.bzl

```diff
@@ -212,3 +206,70 @@ def envoy_dependencies(path = "@envoy_deps//", skip_protobuf_bzl = False, skip_t
     python_deps(skip_targets)
     cc_deps(skip_targets)
     envoy_api_deps(skip_targets)
+
+def com_github_fmtlib_fmt():
+  native.new_http_archive(
+      name = "com_github_fmtlib_fmt",
+      urls = [
+          "https://github.com/fmtlib/fmt/releases/download/4.0.0/fmt-4.0.0.zip",
```

Comment:
What about `repository_locations.bzl`? This was explicitly added to allow site-local redirects of deps.

Reply:
I would prefer to have one way to override dependencies, if possible. This implementation allows deps to be overridden by adding things to `WORKSPACE`, which lets us avoid making local changes to the Envoy source files.